src/models/pieChart.js

```diff
@@ -66,8 +66,8 @@ nv.models.pieChart = function() {
             nv.utils.initSVG(container);
 
             var that = this;
-            var availableWidth = nv.utils.availableWidth(width, container, margin),
-                availableHeight = nv.utils.availableHeight(height, container, margin);
+            var availableWidth = nv.utils.availableWidth(pie.width(), container, margin),
+                availableHeight = nv.utils.availableHeight(pie.height(), container, margin);
```

Comment:
oh really?  that's strange...   pieChart should inherit those options from pie so I thought that should have worked...    regardless if you still have the issue then please submit your fix again and I'll merge it.